pom.xml

```diff
@@ -680,9 +680,9 @@
             </build>
         </profile>
         <profile>
-            <id>hadoop-2.5</id>
+            <id>hadoop-2.6</id>
             <properties>
-                <hadoop.version>2.5.1</hadoop.version>
+                <hadoop.version>2.6.5</hadoop.version>
```

Comment:
why are you eliminating the hadoop-2.5 profile?

Reply:
I didn't see any code specific to 2.5 that 2.3 did not provide?

Reply:
The profile for different versions of Hadoop is both for version specific code as well as for running unit-tests. It is better to keep it and add the hadoop-2.6

Reply:
@chtyim You are asking me to leave the 2.5 profile alone and add the 2.6 profile?

Reply:
Yes, that's correct.